File: app/src/main/java/com/prosabdev/fluidmusic/ui/custom/CustomFastScroller.java
package com.prosabdev.fluidmusic.ui.custom;

import android.content.Context;
import android.util.AttributeSet;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import com.l4digital.fastscroll.FastScroller;

public class CustomFastScroller extends FastScroller {
    public CustomFastScroller(@NonNull Context context) {
        super(context);
    }

    public CustomFastScroller(@NonNull Context context, Size size) {
        super(context, size);
    }

    public CustomFastScroller(@NonNull Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
    }

    public CustomFastScroller(@NonNull Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    @Override
    public void setLayoutParams(@NonNull ViewGroup.LayoutParams params) {
        //super.setLayoutParams(params);
    }

    @Override
    public void setLayoutParams(@NonNull ViewGroup viewGroup) {
        super.setLayoutParams(viewGroup);
    }
}

File: app/src/main/java/com/prosabdev/fluidmusic/ui/custom/CustomFastScroller.kt
package com.prosabdev.fluidmusic.ui.custom

import android.content.Context
import android.util.AttributeSet
import android.view.ViewGroup
import com.l4digital.fastscroll.FastScroller

class CustomFastScroller : FastScroller {
    constructor(context: Context) : super(context)
    constructor(context: Context, size: Size?) : super(context, size)
    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs)
    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(
        context,
        attrs,
        defStyleAttr
    )

    override fun setLayoutParams(params: ViewGroup.LayoutParams) {
        //super.setLayoutParams(params);
    }

    override fun setLayoutParams(viewGroup: ViewGroup) {
        super.setLayoutParams(viewGroup)
    }
}
